src/mon/OSDMonitor.cc

```diff
@@ -2435,8 +2435,8 @@ void OSDMonitor::_prune_update_trimmed(
     version_t first)
 {
   dout(10) << __func__
-           << " first " << first
-           << " last_pinned " << osdmap_manifest.get_last_pinned()
+           << " version " << first
```

Comment:
@changchengx why do you think "version" is better than "first" ? IMHO, "version" is way too general in this context.

Reply:
"version" means something in this context, no? When I'm reading through logs, I always assume "version" would be the current version, never the `first` in whatever context. I don't particularly like this idea.

What's the reasoning here?

Reply:
@tchaikov 
The "version" change here is related to the "committed transaction version".
As @jecluis said here, it would result in ambiguity here. I'll abandon this commit.